awx/main/consumers.py

```diff
@@ -39,6 +43,19 @@ def ws_connect(message):
                 connect_text['user'] = auth_token.user_id
         except AuthToken.DoesNotExist:
             logger.error("auth_token provided was invalid.")
+    else:
+        session_key = request.COOKIES.get(settings.SESSION_COOKIE_NAME, None)
```

Comment:
I think you should talk to @wwitzel3 about what might actually be required here. I *think* session auth is basically transparent? Does the session need to be checked or does the presence of `request.user` imply a valid session as it does in the rest of django?

Reply:
That's a good question, I did not presume session can be transparent to channel also, will check it out.